bower.json

```diff
@@ -18,12 +18,13 @@
     "package.json"
   ],
   "dependencies": {
-    "angular": "^1.4.0",
+    "angular": "^1.5.0",
     "angular-gettext": "~2.0.2",
     "angular-ui-router": "~0.2.13",
     "angular-material": "~1.0.4",
-    "angular-resource": "^1.4.0",
-    "govright-corpus-services": "^1.0.0"
+    "angular-resource": "^1.5.0",
+    "govright-corpus-services": "^1.0.0",
+    "ng-lodash": "~0.4.0"
```

Comment:
This looks like it uses lodash 4.x ... I heard 4.x has some incompatibilities with 3.x. Any idea if this causes trouble?